runtime/src/bank.rs

```diff
@@ -6103,7 +6103,16 @@ impl Bank {
                 .iter()
                 .chain(additional_builtins.unwrap_or(&[]).iter())
             {
-                if builtin.enable_feature_id.is_none() {
+                // The built-in should be added if it has no enable feature ID
+                // and the feature to migrate it to Core BPF is not active.
+                let should_add_builtin = builtin.enable_feature_id.is_none() && {
+                    if let Some(core_bpf_migration) = &builtin.core_bpf_migration_config {
+                        !self.feature_set.is_active(&core_bpf_migration.feature_id)
+                    } else {
+                        true
+                    }
+                };
+                if should_add_builtin {
```

Comment:
We have to figure out how to handle the case where the feature may have been activated, but the migration failed and the program wasn't migrated to BPF.

If this check is left as-is, we're toast!